src/Forms/InvisibleReCaptchaField.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Contributte\ReCaptcha\Forms;
+
+use Contributte\ReCaptcha\ReCaptchaProvider;
+use Nette\Forms\Controls\HiddenField;
+use Nette\Forms\Form;
+use Nette\InvalidStateException;
+use Nette\Utils\Html;
+
+/**
+ * @author Milan Felix Sulc <sulcmil@gmail.com> | Jan Galek <jan.galek@troia-studio.cz>
+ */
+class InvisibleReCaptchaField extends HiddenField
+{
```

Comment:
What about create BaseReCaptcha field? It looks very similar to our other object.

Reply:
yes, but every extend for another object, can be trait ?

Reply:
Oh, I didn't notice. Just keep it. :-) It's ok.